### PR TITLE
Fixed Experience Editor issue

### PR DESCRIPTION
**Pull request by AlekseiRusanov**

### CODE REVIEW

File: src/Project/Wooli/client/client.index.tsx [hunk 1]
```diff
@@ -86,8 +87,18 @@ const initialize = async (jssState?: LayoutServiceData, dictionary?: i18n.Resour
   // if sitecoreData is not available fetch it from SC
   if (!jssState) {
     try {
-      const { language, itemPath } = getRoute(location.pathname);
-
+      const search = queryString.parse(location.search);
+      let language: string, itemPath: string;
+      //sc_itemid means we're in Experience Editor
+      if(search["sc_itemid"]) {
```
Comment (ashishagin) [inline src/Project/Wooli/client/client.index.tsx]: Please insert space

File: src/Project/Wooli/client/client.index.tsx [hunk 2]
```diff
@@ -86,8 +87,18 @@ const initialize = async (jssState?: LayoutServiceData, dictionary?: i18n.Resour
...
+        itemPath = search["sc_itemid"];
+        language = search["sc_lan"];
+      }
+      else {
+        const languageItemPath = getRoute(location.pathname);
```
Comment (ashishagin) [inline src/Project/Wooli/client/client.index.tsx]: You can use destructing with already declared variable

File: src/Project/Wooli/client/client.index.tsx [hunk 3]
```diff
@@ -86,8 +87,18 @@ const initialize = async (jssState?: LayoutServiceData, dictionary?: i18n.Resour
   // if sitecoreData is not available fetch it from SC
   if (!jssState) {
     try {
-      const { language, itemPath } = getRoute(location.pathname);
-
+      const search = queryString.parse(location.search);
+      let language: string, itemPath: string;
+      //sc_itemid means we're in Experience Editor
+      if(search["sc_itemid"]) {
+        itemPath = search["sc_itemid"];
```
Comment (ashishagin) [inline src/Project/Wooli/client/client.index.tsx]: Minor. You could use computed object property names destructuring to get these properties

**Comment by ashishagin** [comment]
Fixes #5

File: src/Project/Wooli/client/client.index.tsx (hunk 1 above)
Comment (AlekseiRusanov) [reply src/Project/Wooli/client/client.index.tsx]: fixed

File: src/Project/Wooli/client/client.index.tsx (hunk 2 above)
Comment (AlekseiRusanov) [reply src/Project/Wooli/client/client.index.tsx]: fixed

File: src/Project/Wooli/client/client.index.tsx (hunk 3 above)
Comment (AlekseiRusanov) [reply src/Project/Wooli/client/client.index.tsx]: fixed

**Pull request merged**